Task 6/Task 6.3/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+    background-color: #333333;
+}
+
+:root {
+    --border-width-before-element: 50px;
+    --border-width-after-element: 46px;
+}
+
+.main-block{
+    border: 5px solid #c2e1f5;
+    text-align: center;
+    margin: auto;
+    margin-top: 70px;
+    background: #88b7d5;
+    min-width: 350px;
+}
+
+.main-block::before {
+    content: "";
+    border-top: var(--border-width-before-element) solid transparent;
+    border-left: var(--border-width-before-element) solid transparent;
+    border-right: var(--border-width-before-element) solid transparent;
+    border-bottom: var(--border-width-before-element) solid #c2e1f5;
+    position: relative;
+    top: -72px;
+}
+
+.main-block::after {
+    content: "";
+    border-top: var(--border-width-after-element) solid transparent;
+    border-left: var(--border-width-after-element) solid transparent;
+    border-right: var(--border-width-after-element) solid transparent;
+    border-bottom: var(--border-width-after-element) solid #88b7d5;
+    position: relative;
+    top: -124px;
+}
```

Comment:
You could save on duplication in 2 ways:
* You can write 1 `border` style, and override `border-bottom`
* You can write styles twice, but combining them like:
```css
.main-block::before,
.main-block::after {
 /* shared stuff */
}

.main-block::before { }

.main-block::after { }
```